.vsts-ci/linux.yml

```diff
@@ -35,6 +35,7 @@ variables:
   POWERSHELL_TELEMETRY_OPTOUT: 1
```

Comment:
@SteveL-MSFT, your last commit had 4 failures in `PowerShell-CI-windows`
Select-String.String actions.Should output a string with the first match highlighted
```powershell
Expected strings to be the same, but they were different.
Expected length: 14
Actual length:   18
Strings differ at index 0.
Expected: '\nhello\nHello\n\n'
But was:  '\r\nhello\r\nHello\r\n\r\n'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Select-String.Tests.ps1: line 91
91:                 $result | Should -Be "`nhello`nHello`n`n"
```
Select-String.String actions.Should output a string with all matches highlighted when AllMatch is used
```powershell
Expected strings to be the same, but they were different.
Expected length: 14
Actual length:   18
Strings differ at index 0.
Expected: '\nhello\nHello\n\n'
But was:  '\r\nhello\r\nHello\r\n\r\n'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Select-String.Tests.ps1: line 104
104:                 $result | Should -Be "`nhello`nHello`n`n"
```
Select-String.String actions.Should output a string with the first match highlighted when SimpleMatch is used
```powershell
Expected strings to be the same, but they were different.
Expected length: 14
Actual length:   18
Strings differ at index 0.
Expected: '\nhello\nHello\n\n'
But was:  '\r\nhello\r\nHello\r\n\r\n'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Select-String.Tests.ps1: line 117
117:                 $result | Should -Be "`nhello`nHello`n`n"
```
Select-String.String actions.Should output a string without highlighting when NoEmphasis is used
```powershell
Expected strings to be the same, but they were different.
Expected length: 14
Actual length:   18
Strings differ at index 0.
Expected: '\nhello\nHello\n\n'
But was:  '\r\nhello\r\nHello\r\n\r\n'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Select-String.Tests.ps1: line 123
123:             $result | Should -Be "`nhello`nHello`n`n"
```